tests/unit/dataactvalidator/test_scripts_loadFSRS.py

```diff
@@ -0,0 +1,73 @@
+from unittest.mock import Mock
+
+import pytest
+
+from dataactcore.models.fsrs import FSRSAward, FSRSSubaward
+from dataactvalidator.scripts import loadFSRS
+from tests.unit.dataactcore.factories import (
+    FSRSAwardFactory, FSRSSubawardFactory)
+
+
+@pytest.yield_fixture()
+def no_award_db(database):
```

Comment:
Sorry @bsweger! The database `fixture` is defined in `tests/unit/conftest.py` as a quick way to set up a database for the unit tests. As noted above, I'm not comfortable with the existing database connection approach, so I've tried to abstract it all away into some sort of "connection". Unfortunately, the `database` fixture is yielding an "InterfaceHolder" while `databaseSession` is yielding a Sqlalchemy `Session`; I'd like to be consistent, but I'm also not sure which approach is a better match for our work. What do you think?